pkpdapp/pkpdapp/templates/index.html

```diff
@@ -6,16 +6,44 @@
 
 {% extends "base/base.html" %}
 {% load static %}
+{% load current_project %}
 
 {% block main_content %}
-	<div class="container">
-		<div class="row py-5">
-			<div class="col">
-				<h1>
-					Welcome to the PKPDApp, a web-based application to model the
-					distribution and effects of drugs.
-				</h1>
-			</div>
-		</div>
-	</div>
+<div class="container">
+<h1>
+  Welcome to the PKPDApp, a web-based application to model the
+	distribution and effects of drugs.
+</h1>
+{% if user.is_authenticated %}
+<p>
+Each user has a number of projects that they have created or are assigned to. Use the drop-down in the upper right corner to select from the list of available

```

Comment:
Not sure this is right now since the drop-down is no longer in the upper right?

Reply:
fixed